### PR TITLE
Fix chord generation

### PR DESCRIPTION
**Pull request by JakubBrojacz**
Fixed chords and tonation classes according to documentation;
Extracted BaseSound class to simplify all music classes;
Fix chords generation according to new music classes interfaces

### CODE REVIEW

File: src/chords_generation.py [hunk 1]
```diff
@@ -20,14 +24,31 @@ def get_tonation_chord(tonation: music.Tonation, sound: music.Sound):
...
+        raise Exception("Not supported meter in chord duration")
+    
+    sounds_at_metrum = get_sounds_at_metrum(sounds, meter)
     chords = [get_tonation_chord(tonation, sound)
               for sound in sounds_at_metrum]
-    
+
     c1 = []
+    # chord_duration = half of meter * how many eigth fits in one meter note
+    chord_duration = (meter[0]/2) * (meter[1]/0.5)
```
Comment (JakubBilski) [inline src/chords_generation.py:49]: shouldn't it be `(meter[1]/2) * (meter[0]/0.5)` ?
I mean the result is the same, but the comment makes more sense

File: src/utils/constants.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+REVERSE_SYMBOLS.update(
+    {
+        'cis': 1,
+        'dis': 3,
+        'fis': 6,
+        'gis': 8,
+        'ais': 10
+    }
+)
+REVERSE_RHYTHMIC_VALUES = {
```
Comment (JakubBilski) [inline src/utils/constants.py:20]: I feel like `RHYTHMIC_VALUES` is misleading. For me it means that the dictionary will return a rhytmic value. Dicts should be generally named VALUES, VALUES_BY_KEY or KEY_TO_VALUE

File: src/main.py [hunk 3]
```diff
@@ -48,10 +53,10 @@ def parse_args():
 
     tonation = tonation_recognition.get_tonation(sounds)
 
-    chords = chords_generation.get_chords(sounds, tonation, meter, sounds[0].timestamp)
+    chords = chords_generation.get_chords(sounds, tonation, (4, 8))
 
     print("Chords:")
-    print("\n".join([f"{chord.timestamp:.3f}: {chord}\t{chord.duration_ms:.3f}" for chord in chords]))  # noqa
+    print("\n".join([f"{chord}\t{chord.duration:.3f}" for chord in chords]))  # noqa
```
Comment (JakubBilski) [inline src/main.py:59]: Why duration, and not duration_ms? Is this not in miliseconds?

File: src/music/base_sound.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+from utils import constants
+
+
+class BaseSound:
```
Comment (JakubBilski) [inline src/music/base_sound.py:4]: Very good idea, using Sounds everywhere was a terrible mess

**Review by JakubBilski — APPROVED** [review]

File: src/chords_generation.py (hunk 1 above)
Comment (JakubBrojacz) [reply src/chords_generation.py:49]: meter 4/8 means there are 4 eigths in one bar. From that chord duration is 2 (half of meter) *  1 (number of eigths in eigth)

File: src/main.py (hunk 3 above)
Comment (JakubBrojacz) [reply src/main.py:59]: Look at documentation

File: src/utils/constants.py (hunk 2 above)
Comment (JakubBrojacz) [reply src/utils/constants.py]: Fixed

File: src/chords_generation.py (hunk 1 above)
Comment (JakubBilski) [reply src/chords_generation.py:49]: Ok, you're right

**Pull request merged**